Sources/Sentry/SentryThreadHandle.cpp

```diff
@@ -240,7 +240,7 @@ namespace profiling {
         if (handle_ == THREAD_NULL) {
             return false;
         }
-        return SENTRY_PROF_LOG_KERN_RETURN(thread_suspend(handle_)) == KERN_SUCCESS;
+        return thread_suspend(handle_) == KERN_SUCCESS;
```

Comment:
We could eventually store the errno and log it after resuming the thread as well.